src/atomate2/aims/jobs/convergence.py

```diff
@@ -0,0 +1,199 @@
+"""Defines the base FHI-aims convergence jobs."""
+
+from __future__ import annotations
+
+import json
+from dataclasses import dataclass, field
+from pathlib import Path
+
+from jobflow import Flow, Maker, Response, job
+from pymatgen.core import Molecule, Structure
+
+from atomate2.aims.jobs.base import BaseAimsMaker
+from atomate2.aims.schemas.task import AimsTaskDoc, ConvergenceSummary
+from atomate2.aims.utils.msonable_atoms import MSONableAtoms
+
+CONVERGENCE_FILE_NAME = "convergence.json"  # make it a constant?
+
+
+@dataclass
+class ConvergenceMaker(Maker):
+    """Defines a convergence workflow with a maximum number of steps.
+
+    A job that performs convergence run for a given number of steps. Stops either
+    when all steps are done, or when the convergence criterion is reached, that is when
+    the absolute difference between the subsequent values of the convergence field is
+    less than a given epsilon.
+
+    Parameters
+    ----------
+    name : str
+        A name for the job
+    maker: .BaseAimsMaker
+        A maker for the run
+    criterion_name: str
+        A name for the convergence criterion. Must be in the run results
+    epsilon: float
+        A difference in criterion value for subsequent runs
+    convergence_field: str
+        An input parameter that changes to achieve convergence
+    convergence_steps: Iterable
+        An iterable of the possible values for the convergence field.
+        If the iterable is depleted and the convergence is not reached,
+        that the job is failed
+    last_idx: int
```

Comment:
Is this needed? Can't you just pass a smaller `convergence_steps` iterable if you want to do fewer steps?

Reply:
removing